### ISSUE TITLE
querynode's error of tickscript 

### ISSUE THREAD

**Issue by CoderOverflow** [influxdata/kapacitor#2026]
Hi,
I cannot get the data from influxdb when I use tickscript sometimes.
![hxg x khp7 u6 14zctkway](https://user-images.githubusercontent.com/29338660/44195465-b7ade680-a16b-11e8-9ab6-5e6e35bdcf31.png)

![vv63 wwq y9r ynem3 w](https://user-images.githubusercontent.com/29338660/44195573-078cad80-a16c-11e8-91f4-e007b878a4a6.png)

![vvbjhewqns9fpfj vpeqh69](https://user-images.githubusercontent.com/29338660/44195652-44f13b00-a16c-11e8-8aa0-3e59cde9811e.png)

It seem wrong that kapacitor generate the wrong SQL . This is the reason why I can't get the data from influxdb correctly.  In this case, How to get rid of the part "AND time >= '2018-08-16T07:13:00Z' AND time < '2018-08-16T07:13:00Z'"

**Comment by arturo-mondelo** [comment]
Yes, .period and .every are the way to write the tick script to run on kapacitor, different from the influx query. (in your case 15m)
